ReactCommon/cxxreact/Portability.h

```diff
@@ -0,0 +1,21 @@
+// Copyright (c) Facebook, Inc. and its affiliates.
+
+// This source code is licensed under the MIT license found in the
+// LICENSE file in the root directory of this source tree.
+
+#pragma once
+
+// Packing uses a push/pop mechanic in MSVC.
```

Comment:
Folly already defines all of these in `folly/Portability.h`, so it's probably better to just use those rather than have another copy of them :)